client/stylesheets/_modal.scss

```diff
@@ -681,10 +682,14 @@
 
 /*~~ Button End ~~*/
 
-.modAddListingPageThree, .modAddListingPageFive, .modAddListingPageSix {
+.modAddListingPageThree, .modAddListingPageFive {
```

Comment:
Selector `modAddListingPageThree` should be written in lowercase with hyphens<br>Selector `modAddListingPageFive` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line